inc/thorvg.h

```diff
@@ -757,28 +757,67 @@ class TVG_API RadialGradient final : public Fill
     /**
      * @brief Sets the radial gradient bounds.
      *
-     * The radial gradient bounds are defined as a circle centered in a given point (@p cx, @p cy) of a given radius.
+     * The radial gradient bounds are defined as a circle centered in a given point (@p cx, @p cy) of a given radius @p r.
      *
      * @param[in] cx The horizontal coordinate of the center of the bounding circle.
      * @param[in] cy The vertical coordinate of the center of the bounding circle.
-     * @param[in] radius The radius of the bounding circle.
+     * @param[in] r The radius of the bounding circle.
      *
-     * @return Result::Success when succeed, Result::InvalidArguments in case the @p radius value is zero or less.
+     * @return Result::Success when succeed, Result::InvalidArguments in case the @p r value is zero or less.
      */
-    Result radial(float cx, float cy, float radius) noexcept;
+    Result radial(float cx, float cy, float r) noexcept;
```

Comment:
I thought that we want to have both setters, don't we?
otherwise the user have to always specify the values of fx, fy, fr. we can not use def values, since for fx and fy these are cx and cy.

Reply:
Thorvg maintains optimal number of APIs. I believe that the new API won't cause any troubles for users. we can address the guide in doc.

Reply:
@hermet please check now. I've changed the apis as you suggested. I kept the beta_api tag, added a comment what values to use for the focal point.